src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -41,7 +42,9 @@
  *
  * @author depristo
  */
-public final class CommonInfo {
+public final class CommonInfo implements Serializable {
```

Comment:
The options are to either change it to `Map<String, Serializable>` which could potentially be a breaking change for people using serializable attributes, or to add a doc string saying "these must be serializable or serialization will fail"  which will not break any ones code, but means that serialization might fail.  We were leaning towards the "document and force it onto the user" policy since it seems controversial to potentially break anyone's code.